### PR TITLE
New package: libltc

### PR DESCRIPTION
**Pull request by bluntphenomena**
needed for x42-plugins

### CODE REVIEW

File: srcpkgs/libltc/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'libltc'
+pkgname=libltc
+version=1.3.1
+revision=1
+build_style=gnu-configure
+short_desc="Linear/Logitudinal Time Code (LTC) Library"
+maintainer="bluntphenomena <gregwyd@gmail.com>"
+license="GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/libltc/template]: needs spdx identifier

Comment (bluntphenomena) [reply srcpkgs/libltc/template]: ok how do i change this correctly without making too much commits or pull requests?
i cant figure out how to overwrite this commit after i made already the pull request

**Comment by maxice8** [comment]
On Thu, Apr 04, 2019 at 04:13:17PM -0700, bluntphenomena wrote:
> bluntphenomena commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,23 @@
> +# Template file for 'libltc'
> +pkgname=libltc
> +version=1.3.1
> +revision=1
> +build_style=gnu-configure
> +short_desc="Linear/Logitudinal Time Code (LTC) Library"
> +maintainer="bluntphenomena <gregwyd@gmail.com>"
> +license="GPL-3.0"
> 
> ok how do i change this correctly without making too much commits or pull requests?
> i cant figure out how to overwrite this commit after i made already the pull request
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10574#discussion_r272400324

1. make change
2. git add <path to file>
3. git commit --amend

**Comment by maxice8** [comment]
I recomend reading up on 

[git commit --amend](https://git-scm.com/docs/git-commit)
[git rebase --interactive](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
[git remote](https://git-scm.com/docs/git-remote)

Those are used a lot when contributing to Void Linux

**Pull request closed without merging**